.github/workflows/CD.yml

```diff
@@ -0,0 +1,42 @@
+name: Build and Release k-NN
+on:
+  push:
+    tags: 
+      - v*
```

Comment:
So, this will basically trigger whenever we create a tag that starts in v for releases correct?

Reply:
Yes